provider/provider.go

```diff
@@ -79,6 +79,7 @@ func DatabricksProvider() *schema.Provider {
 			"databricks_group":                       scim.ResourceGroup(),
 			"databricks_group_instance_profile":      aws.ResourceGroupInstanceProfile(),
```

Comment:
please also add a deprecation message for `databricks_group_instance_profile` in resource definition and documentation to use `databricks_group_role`

Reply:
Oh, did not know that you have a plan to deprecate group_instance_profile. May I know which version I should add to deprecation message

```
Please migrate to `databricks_group_role`. This resource will be removed in v1.2.?
```

Reply:
Just `Please migrate to `databricks_group_role`.` is enough. We'll remove it in 2.0, because we're GA now and that's the consistency guarantees we have ;)